http-gateway/web/i18n/template.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "defaultMessage": "API",
```

Comment:
I need an explanation here why, what is the reason it cannot be a part of it? The reason why I decided to keep it in, so that we can always generate the final json file usable for the application anytime new .po files are pushed from the translation server.